### PR TITLE
E2e tests rack resources

### PR DESCRIPTION
**Pull request by Twsouza**
### What is the feature/fix?

Create a new set of CI Tests to test apps resources, apps resources and timer. 

The `example/httpd` will only include the minimum convox.yaml possible. Created a new `examples/full-convox-yaml` to include more parameters to be used on the new CI tests.

It will increase the test timer from 02:30~03:00 to 01:45 ~02:00.

### Does it has a breaking change?

No.

### How to use/test it?

N/A

### Checklist
- [N/A] New coverage tests
- [ ] Unit tests passing
- [ ] E2E tests passing
- [ ] E2E downgrade/update test passing
- [N/A] Documentation updated
- [ ] No warnings or errors on Deepsource/Codecov

### CODE REVIEW

**Review by yuriadams — APPROVED** [review]

File: provider/aws/formation/rack.json [hunk 1]
```diff
@@ -2943,6 +2943,7 @@
                 "iam:CreatePolicyVersion",
                 "iam:CreateRole",
                 "iam:CreateUser",
+                "iam:DeleteAccessKey",
```
Comment (nightfury1204) [inline provider/aws/formation/rack.json:2946]: is this intended?

Comment (Twsouza) [reply provider/aws/formation/rack.json:2946]: yes, while testing I found an issue on deleting the rack resources. The iam permission is needed to delete the resources.

**Review by nightfury1204 — DISMISSED** [review]

**Comment by Twsouza** [comment]
@nightfury1204 can you review it again plz?

**Review by nightfury1204 — COMMENTED** [review]
Looks good. Did you run e2e on this branch?

**Comment by Twsouza** [comment]
@nightfury1204 yes

**Pull request closed without merging**